yogstation/code/modules/mob/living/simple_animal/hostile/floor_cluwne.dm

```diff
@@ -93,6 +94,11 @@ GLOBAL_VAR_INIT(floor_cluwnes, 0)
 	var/turf/T = get_turf(current_victim)
 	if(prob(5))//checks roughly every 20 ticks
 		if(current_victim.stat == DEAD || current_victim.dna.check_mutation(CLUWNEMUT) || is_type_in_typecache(get_area(T), invalid_area_typecache) || !is_station_level(current_victim.z))
+			for(var/obj/structure/closet/hiding_spot in orange(7,src))
```

Comment:
This doesn't actually check if the victim is actually inside the closet

Reply:
this check for if they're in a locker doesnt check if theyre actually in a locker.